docs/main/BasicClient/Hero.md

```diff
@@ -257,6 +255,33 @@ Close a single Tab. The first opened Tab will become the focused tab.
 
 Alias for [Tab.close()](/docs/hero/basic-client/tab#close)
 
+
+### hero.detach *(elementOrResource, options?)* {#detach}
+
+Detaches an element or resource from AwaitedDOM and converts it into DetachedDOM. You can supply an optional name in the 2nd argument to store the element in [hero.detachedElements](/docs/hero/basic-client/hero#detachedElements) for later use.
+
+```js
+const hero = new Hero();
+await hero.goto('https://ulixee.org');
+await hero.detach('title', hero.querySelector('h1'));
+const when = await hero.detachedElements.get('title');
```

Comment:
Copy paste error with the name here

Reply:
What is the error?

Reply:
No error, but the name of "when" seemed like it was copied from the example where you saved "time". It threw me off here.